instrumentation/spring-rabbit/src/test/java/brave/spring/rabbit/ITSpringRabbitTracing.java

```diff
@@ -131,4 +135,26 @@
     assertThat(consumerReporter.takeLocalSpan().name())
       .isEqualTo("on-message");
   }
+
+  @Test public void producerSampler() {
```

Comment:
inlining these tests saves 15s each on my laptop as the overall container bootstrap seems to have 15s overhead